docs/scos/dev/technology-partner-guides/202108.0/payment-partners/braintree/braintree-performing-requests.md

```diff
@@ -68,7 +68,8 @@ In order to perform the necessary requests in the project based on Spryker Comme
 | FACADE METHOD | PARAMETER | RETURN | DESCRIPTION |
 | --- | --- | --- | --- |
 |  `saveOrderPayment` | `QuoteTransfer``CheckoutResponseTransfer` | void | Saves order payment method data according to quote and checkout response transfer data. |
-|  `preCheckPayment` |  `QuoteTransfer` |  `BraintreeTransactionResponseTransfer` | Sends pre-authorize payment request to Braintree gateway to retrieve transaction data. Checks that form data matches transaction response data. |
+|  `preCheckPayment` |  `QuoteTransfer` |  `BraintreeTransactionResponseTransfer` | Sends pre-authorize payment request to Braintree gateway to retrieve transaction data. Checks that form data matches transaction response data. **This method is deprecated**: use `isQuotePaymentValid()` instead. |
+|  `isQuotePaymentValid` |  `QuoteTransfer` |  `BraintreeTransactionResponseTransfer` | Checks that quote has required data to perform a Braintree payment transaction. |
```

Comment:
don't we need to add new facade methods to this list?

Reply:
There were 2 facade methods introduced, `isQuotePaymentValid()` and `executeCheckoutPostSaveHook`. They were both added